kitsune/questions/jinja2/questions/question_list.html

```diff
@@ -169,11 +169,11 @@ <h1 class="sumo-page-heading {% if not logo %}no-logo{% endif %}">
           {% for question in questions.object_list %}
             <article id="question-{{ question.id }}" class="forum--question-item">
               <ul class="forum--question-item-side">
-                <li class="thread-solved {% if question.is_solved %}highlighted{% endif %}">
+                <li {% if question.is_solved %} class="thread-solved highlighted" {% endif %}>
                   <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
                     <path stroke="currentColor" stroke-width="2" d="M20 6L9 17l-5-5" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"/>
                   </svg>
-                  <span class="hover-label">{{ _('Solved') }}</span>
+                  <span class="hover-label">{% if question.is_solved %} {{ _('Solved') }} {% else %} {{ _('Not solved') }} {% endif %}</span>
                 </li>
                 <li class="thread-contributed {% if question.is_contributor(user) %}highlighted{% endif %}">
```

Comment:
this might need to be changed too - "contributed" state is off as well currently